docs/developer-handbook.md

```diff
@@ -577,26 +503,25 @@ This will run the `watch` command in `gulpfile.js`. As a result, whenever you
 make a change to the project styles it will automatically copy over into the
 `scss` folder which Storybook uses in `carbon-components-angular`.
 
-### Pointing NPM dependency of `carbon-components` right to the source code
+### Pointing NPM dependency of `@carbon/styles` right to the source code
 
 Though above approach is the most straightforward, it involves an overhead of
-having to run build process at `carbon-components`, in addition to one at
-framework variant repo, upon every Sass code change.
+having to run build process at `@carbon/styles`, in addition to one at framework
+variant repo, upon every Sass code change.
 
-To avoid such overhead, you can point NPM dependency of `carbon-components`
-right to the source code, though there is a caveat that our future change to
-directory structure, etc. may make such steps no longer work. Here are the
-steps:
+To avoid such overhead, you can point NPM dependency of `@carbon/styles` right
+to the source code, though there is a caveat that our future change to directory
+structure, etc. may make such steps no longer work. Here are the steps:
 
 ```sh
-> cd /path/to/carbon-components-angular/node_modules/carbon-components
+> cd /path/to/carbon-components-angular/node_modules/@carbon/styles
 > mv scss scss.orig
-> ln -s /path/to/carbon-components/src scss
+> ln -s /path/to/@carbon/styles/src scss
```

Comment:
```suggestion
> ln -s /path/to/@carbon/styles/scss scss
```

We don't have a `src` folder in styles, should this point to the `scss` folder?

Reply:
Yes, thanks for the catch!